src/TestRunnerContext.php

```diff
@@ -120,12 +126,21 @@ public function afterRunTests(AfterScenarioScope $scope)
      */
     public function createWorkingDirectory()
     {
-        $this->workingDirectory = tempnam(sys_get_temp_dir(), 'behat-test-runner');
-        $this->filesystem->remove($this->workingDirectory);
-        $this->filesystem->mkdir($this->workingDirectory . '/features/bootstrap', 0770);
+        if (empty($this->workingDirectory)) {
+            $this->workingDirectory = sys_get_temp_dir() . '/behat-test-runner';
+        }
+
+        $featuresDirectory = $this->workingDirectory . '/features/bootstrap';
+
+        if (!$this->filesystem->exists($featuresDirectory)) {
+            $this->filesystem->mkdir($this->workingDirectory . '/features/bootstrap', 0770);
+        }
 
         $this->documentRoot = $this->workingDirectory .'/document_root';
-        $this->filesystem->mkdir($this->documentRoot, 0770);
+
+        if (!$this->filesystem->exists($featuresDirectory)) {
```

Comment:
You are checking the existence of the `$featuresDirectory`, but creating `$this->documentRoot`. That's a bug.
It should be like
```php
if (!$this->filesystem->exists($this->documentRoot)) {
```

Reply:
oopsy :D

Reply:
fixed